doc/nrf/samples/wifi_zephyr.rst

```diff
@@ -0,0 +1,48 @@
+.. _wifi_zephyr_samples:
+
+Wi-Fi: Zephyr networking samples
+################################
+
+In addition to nRF Connect SDK samples, it is also possible to run selected Zephyr networking
+samples with Wi-Fi.
+The following list specifies samples that are currently supported with the nRF700x driver:
```

Comment:
same comment here.

Reply:
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/protocols/wifi/index.html
(Wi-Fi driver is the name, at least in this page)

Reply:
I was inspired by "Working with nRF700x" section where it referred "nRF700x driver": https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/device_guides/working_with_nrf/nrf70/developing/constrained.html#id9
But I can switch to Wi-Fi driver if it's more commonly used.

Reply:
Not sure if **nRF700X Wi-Fi driver** could be the alternative as used in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/drivers/nrf700x.html.

Reply:
I am considering us phasing out the nRF700X term. Marketing has already switched to mostly used nRF70 Series, @richabp. Keep Wi-Fi driver for now, IMHO.

Reply:
@ioannisg Sure!